src/components/NcActionCheckbox/NcActionCheckbox.vue

```diff
@@ -118,7 +118,7 @@ export default {
 	},
 
 	methods: {
-		checkInput(event) {
+		checkInput(/* event */) {
```

Comment:
Is this necessary for typescript? Just asking out of curiosity.

Reply:
I can also remove it, but it is unused and otherwise ESLint will complain about unused parameter

Reply:
What would you prefer? Let it there as a comment or remove the unused argument?

Reply:
I don't mind. You can leave it as a comment.